extensions/amp-story/1.0/amp-story.css

```diff
@@ -232,6 +243,24 @@ amp-story-page[active] {
   }
 }
 
+.i-amphtml-amp-story-sidebar-mask {
+  position: fixed !important;
+  top: 0 !important;
+  left: 0 !important;
+  width: 100vw !important;
+  height: 100vh !important;
```

Comment:
Since the sidebar is a direct child of `amp-story`, you can use `position: absolute; top: 0; bottom: 0; width: 100%; height: 100%;`. We try to position everything after `amp-story`, for if we ever need to display several stories in the same page.

Also, please add some `!important` statements at the end of all the CSS rules, so publishers can't override them :))